dimagi/urls.py

```diff
@@ -69,6 +69,12 @@ def get_robots():
         pillar.mobile_data_collection, name="mobile_data_collection"),
     url(r'^blog/mobile-data-collection-introduction/$',
         redirect.page('mobile_data_collection')),
+    url(r'^blog/mobile-data-collection-standards/$',
+        redirect.page('mobile-data-collection-mobile-survey-design')),
```

Comment:
hmm...did you test this? I don't think this will work as this is specifically a blog post...there is an example of this in redirect.py

Reply:
I've updated in redirect.py